### ISSUE TITLE
[Parity] Dungeons Dirt

### ISSUE THREAD

**Issue by nikolayVR06** [BedrockTweaks/Files#254]
**Dungeons dirt**

Sides of the grass block are green, like it is applying folliage color to all side of the block

![изображение_2024-08-10_202457465](https://github.com/user-attachments/assets/29f90a21-e008-41ed-bb31-187d43e182b8)

 - OS: Windows 10
 - Minecraft Version: [e.g 1.21.2]

All of the dirt types seem to work normally, except for the grass block
![image](https://github.com/user-attachments/assets/ede35980-cca2-4aca-a0bf-accd46085a58)

**Comment by SquatchHunter** [comment]
I had a dig around at this one, idk what is exactly going wrong.
I can narrow it down to the "overlay_colour" in terrain_texture.json is being applied to the whole texture, not just the grass like it does in vanilla

swapping the .png file for a .tga file like the _official_ Mojang Sample has it does not fix this either, so if someone with more texture experience wants to tackle this, they may have more luck

**Comment by PAPISOP** [comment]
I also tried fixing it by first adding the `terrain_texture.json` and removing all instances of `overlay_color` in grass so it seems to fix it in plains biome but in other biomes it just becomes plains variant which is not correct

Then, i tried to remove `grass_side.png` and added `grass_side.tga` first without any modifications, didn't work, then i added the custom grass side texture to the tga file, but still didn't fix it

In conclusion, I think this is a **Minecraft Bug**, but again if you know how to fix it then try it out

**Comment by SquatchHunter** [comment]
that was the extent of my testing so if someone can confirm it then thats atleast _progress_ in identifing the issue

**Comment by PAPISOP** [comment]
This bug should be fixed by my pr! The bug itself was pretty weird and I have to do weird stuff with the help of @Chi-a11y to get it working
